tests/catalog/test_models.py

```diff
@@ -266,17 +268,32 @@ def test_long_name(self):
         It may create problems for tag with long name.
         """
         name = 'Имя ' * 50
-        group = catalog_models.MockTagGroup.objects.first()
+        group = catalog_models.MockTagGroup.objects.create(name='Some group')
         try:
             tag = catalog_models.MockTag.objects.create(group=group, name=name)
             self.assertLessEqual(len(tag.slug), catalog.models.Tag.SLUG_MAX_LENGTH)
         except DataError as e:
             self.assertTrue(False, f'Tag has too long name. {e}')
 
+    # @todo #302:30m  Process more special symbols for slugs.
+    @unittest.expectedFailure
     def test_slugify_conflicts(self):
         slugs = [
             catalog_models.MockTag.objects.create(name=name).slug
-            for name in ['11 A', '1/1 A', '1 1 A']
+            for name in ['11 A', '1/1 A', '1 1 A', '1.1 A', '1-1 A', '1_1 A']
```

Comment:
Let's create a separate skipped test (e.g. `test_slugify_punctuation_conflicts`) for the set, that led to the conflict to preserve testing of the set that has no such conflicts `11 A, 1/1 A, 1 1 A`

This will help to avoid regressions if we break the slugify for non-conflict set. In common i'd prefer stick to such rule: new tests for new bugs. Up to you